CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```
@@ -141,6 +174,114 @@ stages:
+          crowdstrike.idp_policy_rule_trigger: "{{ parsed_event.message.event.IdpPolicyRuleTrigger }}"
+          crowdstrike.pattern_id: "{{ parsed_event.message.event.PatternId }}"
+          crowdstrike.previous_privileges: "{{ parsed_event.message.event.PreviousPrivileges }}"
+          destination.domain: "{{ parsed_event.message.event.TargetEndpointHostName }}"
+          event.end: "{{ parse_event_end_date.datetime }}"
+          event.reason: "{{ parsed_event.message.event.Objective }}"
+          event.reference: "{{ parsed_event.message.event.FalconHostLink }}"
+          event.severity: "{{ parsed_event.message.event.Severity }}"
+          event.start: "{{ parse_event_start_date.datetime }}"
+          event.url: "{{ parsed_event.message.event.FalconHostLink }}"
+          host.id: "{{ parsed_event.message.event.SourceEndpointAccountObjectGuid }}"
+          host.name: "{{ parsed_event.message.event.SourceEndpointHostName }}"
+          rule: "{{ parsed_event.message.event.IdpPolicyRuleAction }}"
+          rule.name: "{{ parsed_event.message.event.IdpPolicyRuleName }}"
+          source.geo.country_iso_code: "{{ parsed_event.message.event.LocationCountryCode }}"
+          source.ip: "{{ parsed_event.message.event.SourceEndpointIpAddress }}"
+          threat.tactic.name: "{{ parsed_event.message.event.Tactic }}"
+          threat.technique.name: "{{ parsed_event.message.event.Technique }}"
+          user.domain: "{{ parsed_event.message.event.SourceAccountDomain }}"
+          user.email: "{{ parsed_event.message.event.SourceAccountUpn }}"
+          user.id: "{{ parsed_event.message.event.SourceAccountObjectSid }}"
+          user.name: "{{ parsed_event.message.event.SourceAccountName }}"
+          user.target.domain: "{{ parsed_event.message.event.TargetAccountDomain }}"
+          user.target.email: "{{ parsed_event.message.event.TargetAccountUpn }}"
+          user.target.id: "{{ parsed_event.message.event.TargetAccountObjectSid }}"
+          user.target.name: "{{ parsed_event.message.event.TargetAccountName }}"
+
+  identityprotectionevent:
+    actions:
+      - set:
+          crowdstrike.incident_description: "{{ parsed_event.message.event.IncidentDescription }}"
+          crowdstrike.incident_type: "{{ parsed_event.message.event.IncidentType }}"
+          crowdstrike.severity_name: "{{ parsed_event.message.event.SeverityName }}"
+          event.end: "{{ parse_event_end_date.datetime }}"
+          event.reference: "{{ parsed_event.message.event.FalconHostLink }}"
+          event.severity: "{{ parsed_event.message.event.Severity }}"
+          event.start: "{{ parse_event_start_date.datetime }}"
+          event.url: "{{ parsed_event.message.event.FalconHostLink }}"
+          host.name: "{{ parsed_event.message.event.EndpointName }}"
+          source.ip: "{{ parsed_event.message.event.EndpointIp }}"
+          user.name: "{{ parsed_event.message.event.UserName }}"
```

I suggest splitting `UserName` when its value matches the pattern `DOMAIN\\Username`.
```suggestion
          user.name: "{{ parsed_event.message.event.UserName }}"
      - set:
          user.domain: "{{ parsed_event.message.event.UserName.split('\\')[0] }}"
          user.name: "{{ parsed_event.message.event.UserName.split('\\')[1] }}"
      filter: "{{ '\\' in parsed_event.message.event.UserName }}"
```